Docs/source/install/hpc/lumi.rst

```diff
@@ -22,64 +22,112 @@ If you are new to this system, **please see the following resources**:
-   :language: bash
-   :caption: You can copy this file from ``Tools/machines/lumi-csc/lumi_warpx.profile.example``.
+.. code-block:: bash
 
+   cp $HOME/src/warpx/Tools/machines/lumi-csc/lumi_warpx.profile.example $HOME/lumi_warpx.profile
 
-We recommend to store the above lines in a file, such as ``$HOME/lumi_warpx.profile``, and load it into your shell after a login:
+.. dropdown:: Script Details
+   :color: light
+   :icon: info
+   :animate: fade-in-slide-down
 
-.. code-block:: bash
+   .. literalinclude:: ../../../../Tools/machines/lumi-csc/lumi_warpx.profile.example
+      :language: bash
+
+Edit the 2nd line of this script, which sets the ``export proj=""`` variable using a text editor
+such as ``nano``, ``emacs``, or ``vim`` (all available by default on
+LUMI login nodes).
+
+.. important::
+
+   Now, and as the first step on future logins to LUMI, activate these environment settings:
 
-   source $HOME/lumi_warpx.profile
+   .. code-block:: bash
+
+      source $HOME/lumi_warpx.profile
 
-And since LUMI does not yet provide a module for them, install c-blosc and ADIOS2:
+Finally, since LUMI does not yet provide software modules for some of our dependencies, install them once:
 
 .. code-block:: bash
 
-   export CMAKE_PREFIX_PATH=${HOME}/sw/lumi/gpu/c-blosc-1.21.1:$CMAKE_PREFIX_PATH
-   export CMAKE_PREFIX_PATH=${HOME}/sw/lumi/gpu/adios2-2.8.3:$CMAKE_PREFIX_PATH
+   bash $HOME/src/warpx/Tools/machines/lumi-csc/install_dependencies.sh
+   source $HOME/sw/lumi/gpu/venvs/warpx-lumi/bin/activate
+
+.. dropdown:: Script Details
+   :color: light
+   :icon: info
+   :animate: fade-in-slide-down
+
+   .. literalinclude:: ../../../../Tools/machines/lumi-csc/install_dependencies.sh
+      :language: bash
 
-   # c-blosc (I/O compression)
-   git clone -b v1.21.1 https://github.com/Blosc/c-blosc.git src/c-blosc
-   rm -rf src/c-blosc-build
-   cmake -S src/c-blosc -B src/c-blosc-build -DBUILD_TESTS=OFF -DBUILD_BENCHMARKS=OFF -DDEACTIVATE_AVX2=OFF -DCMAKE_INSTALL_PREFIX=${HOME}/sw/lumi/gpu/c-blosc-1.21.1
-   cmake --build src/c-blosc-build --target install --parallel 16
 
-   # ADIOS2
-   git clone -b v2.8.3 https://github.com/ornladios/ADIOS2.git src/adios2
-   rm -rf src/adios2-build
-   cmake -S src/adios2 -B src/adios2-build -DADIOS2_USE_Blosc=ON -DADIOS2_USE_Fortran=OFF -DADIOS2_USE_Python=OFF -DADIOS2_USE_ZeroMQ=OFF -DCMAKE_INSTALL_PREFIX=${HOME}/sw/lumi/gpu//adios2-2.8.3
-   cmake --build src/adios2-build --target install -j 16
+.. _building-lumi-compilation:
 
-Then, ``cd`` into the directory ``$HOME/src/warpx`` and use the following commands to compile:
+Compilation
+-----------
+
+Use the following :ref:`cmake commands <building-cmake>` to compile:
 
 .. code-block:: bash
 
    cd $HOME/src/warpx
-   rm -rf build
-
-   cmake -S . -B build -DWarpX_DIMS="1;2;3" -DWarpX_COMPUTE=HIP -DWarpX_PSATD=ON -DWarpX_QED_TABLE_GEN=ON
-   cmake --build build -j 16
+   rm -rf build_lumi
```

Comment:
I usually take a `build_<computer>` name if the home directory is shared between different computers. This is for instance the case at NERSC and OLCF.

Maybe CSC the same, but otherwise you can just call it `build` :)